api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -48,6 +54,40 @@ message RedisProxy {
     bool enable_hashtagging = 2;
   }
 
-  // Network settings for the connection pool to the upstream cluster.
+  // Network settings for the connection pool to the upstream clusters.
   ConnPoolSettings settings = 3 [(validate.rules).message.required = true];
+
+  message PrefixRoute {
```

Comment:
nit: I might just embed this message inside `PrefixRoutes` and call it `Route` ? WDYT?